bootstrap/src/index.ts

```diff
@@ -35,6 +37,21 @@ const withStatusCode: Middleware = async (execute) => async (input) => {
   return { ...rest, statusCode, data }
 }
 
+export const withRateLimit: Middleware = async (execute) => async (input) => {
+  const totalReqPerMin = rateLimit.getTroughput(rateLimit.Interval.MINUTE)
+  const participantReqPerMin = rateLimit.getTroughput(rateLimit.Interval.MINUTE, input)
+  const cost = rateLimit.getParticipantCost(participantReqPerMin)
+  const maxReqPerMin = rateLimit.getMaxReqAllowed(
+    totalReqPerMin.length + 1,
+    participantReqPerMin.length + 1,
```

Comment:
If the request succeed, we could assume that the request will be 1 more than the current one.